utility/src/main/java/org/oppia/android/util/statusbar/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+"""
+Utility to change the color of Status Bar.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "status_bar_color",
+    srcs = [
+        "StatusBarColor.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        "//third_party:androidx_appcompat_appcompat",
+    ]
```

Comment:
```suggestion
    ],
```

Reply:
As for now, this will fix it. I will create a bazel lint guide in our wiki this week.

Reply:
Also, by the time you can setup this, if you are not on windows - https://formulae.brew.sh/formula-linux/buildifier

Reply:
I also used buildifier earlier, but it was just mentioning that there is a some lint failure in both the BUILD.bazel files I have changed. It was not actually pointing out the cause of failure.

Reply:
buildifier has a different type of output while sending error, like `#reformat`. Will try to cover all possible cases in wiki page. 
by the time, if you want to check what command we run in checks, you can look at this script - https://github.com/oppia/oppia-android/blob/develop/scripts/buildifier_lint_check.sh

Reply:
@ARJUPTA you can run buildifier on the file itself to let buildifier format the file.